pkg/backend/display/watch.go

```diff
@@ -0,0 +1,118 @@
+// Copyright 2016-2018, Pulumi Corporation.
```

Comment:
Using 2018 for new code written at the end of 2019 still bugs me, FWIW.

Reply:
Fixed - though we really need to do this more holistically.